frontend/src/pages/forms/Form.jsx

```diff
@@ -117,11 +117,7 @@ const Form = () => {
         <div className="form-title">
           <h1>Please submit your leverage details</h1>
         </div>
-        {alertMessage && (
-          <p className="error-message form-alert">
-            {alertMessage} <AlertHexagon className="form-alert-hex" />
-          </p>
-        )}
+        {alertMessage && notify(alertMessage, "error1")}
```

Comment:
I see it currently duplicates errors on a screen. I believe you can remove `alertMessage` state and keep only `error`, and in place of setAlertMessage use `setError`.

Also, notify should be triggered in useEffect, based on `[error]` and after it is shown, error should be cleared, otherwise I afraid that each rerender will cause this toast to show

Reply:
I just realised, you don't need state at all, you can remove both `error` and `alertMessage` states and use `notify` directly inside`handleTransaction` (in two places) instead of `setError.
and inside `handleSubmit` instead of `setAlertMessage`